### PR TITLE
improving functions name

### PR DESCRIPTION
**Pull request by fmoral2**
#### Proposed Changes ####

- The idea is to standardize some names across framework with more idiomatic golang patterns ( shorter and concise naming , also avoiding at most stuff like (gets and set ) have only left get when its totally appropriated like one the get nodes and get pods and adding  more meaningful names 

#### Testing ####

Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

2. Verify code lint; we should not have errors.

3. Update the documentation if needed.

4. Update makefile and docker run if adding new tests.

5. Run your tests at least 4 times with all configurations needed and possible.

6. If needed test with different os types.

### CODE REVIEW

File: entrypoint/upgradecluster/upgrademanual_test.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ var _ = Describe("Test:", func() {
 	It("Checks Node Status pos upgrade and validate version", func() {
```
Comment (mdrahman-suse) [inline entrypoint/upgradecluster/upgrademanual_test.go:70]: typo: post

File: factory/cluster.go [hunk 2]
```diff
@@ -11,13 +11,13 @@ import (
...
-func AddCluster(g GinkgoTInterface) *Cluster {
+// BuildCluster returns a singleton cluster with all terraform config and vars
+func BuildCluster(g GinkgoTInterface) *Cluster {
 	once.Do(func() {
 		var err error
-		cluster, err = newCluster(g)
+		_, err = newCluster(g)
 		if err != nil {
-			err = shared.ReturnLogError("error getting cluster: %w\n", err)
+			err = shared.ReturnLogError("error getting c: %w\n", err)
```
[the remark this answers is not in the document]
Comment (fmoral2) [reply factory/cluster.go:20]: typo, tks

File: pkg/assert/host.go [hunk 3]
```diff
@@ -9,17 +9,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// CheckComponentCmdHost runs a command on the host and asserts that the value received contains the specified substring
+// CheckCmdHost runs a command on the host and asserts that the value received contains the specified substring
```
Comment (ShylajaDevadiga) [inline pkg/assert/host.go:12]: ExecCmdOnHost or RunCmdOnHost

File: pkg/assert/node.go [hunk 4]
```diff
@@ -38,10 +38,10 @@ func assertVersion(c customflag.FlagConfig) NodeAssertFunc {
 
 // assertCommit returns the NodeAssertFunc for asserting commit
 func assertCommit(c customflag.FlagConfig) NodeAssertFunc {
-	product, err := shared.GetProduct()
+	product, err := shared.FetchProduct()
 	Expect(err).NotTo(HaveOccurred(), "error getting product: %v", err)
 
-	commit, err := shared.GetProductVersion(product)
+	commit, err := shared.ProductVersion(product)
```
Comment (ShylajaDevadiga) [inline pkg/assert/node.go:44]: FetchProductVersion to align with FetchProduct

File: pkg/assert/node.go [hunk 5]
```diff
@@ -72,15 +72,15 @@ func NodeAssertReadyStatus() NodeAssertFunc {
 	}
 }
 
-// CheckComponentCmdNode runs a command on a node and asserts that the value received
+// CheckCmdNode runs a command on a node and asserts that the value received
 // contains the specified substring.
-func CheckComponentCmdNode(cmd, assert, ip string) error {
+func CheckCmdNode(cmd, assert, ip string) error {
```
Comment (ShylajaDevadiga) [inline pkg/assert/node.go:77]: Same as above Run or Exec

File: pkg/assert/node.go [hunk 6]
```diff
@@ -72,15 +72,15 @@ func NodeAssertReadyStatus() NodeAssertFunc {
...
 	if cmd == "" || assert == "" {
 		return shared.ReturnLogError("cmd and/or assert should not be sent empty")
 	}
 
 	Eventually(func(g Gomega) {
-		res, err := shared.RunCommandOnNode(cmd, ip)
+		res, err := shared.RunCmdNode(cmd, ip)
```
Comment (ShylajaDevadiga) [inline pkg/assert/node.go:83]: Suggesting RunCmdOnNode to be more clear.

File: pkg/template/helper.go [hunk 7]
```diff
@@ -56,11 +56,11 @@ func executeTestCombination(v VersionTestTemplate) error {
 	return nil
 }
 
-// AddTestCases returns the test case based on the name to be used as customflag.
-func AddTestCases(names []string) ([]testCase, error) {
-	var testCases []testCase
+// TestCases returns the test case based on the name to be used as customflag.
+func TestCases(names []string) ([]testCase, error) {
+	var tcs []testCase
```
Comment (ShylajaDevadiga) [inline pkg/template/helper.go:61]: Let's retain testCases

File: pkg/template/helper.go [hunk 8]
```diff
@@ -56,11 +56,11 @@ func executeTestCombination(v VersionTestTemplate) error {
...
 
-	testCase := map[string]testCase{
+	tc := map[string]testCase{
```
Comment (ShylajaDevadiga) [inline pkg/template/helper.go:63]: same testCase/testcase

File: factory/cluster.go [hunk 9]
```diff
@@ -27,7 +27,7 @@ func AddCluster(g GinkgoTInterface) *Cluster {
 
 // newCluster creates a new cluster and returns his values from terraform config and vars
 func newCluster(g GinkgoTInterface) (*Cluster, error) {
-	terraformOptions, varDir, err := addTerraformOptions()
+	opts, varDir, err := terraformOptions()
```
[the remark this answers is not in the document]
Comment (fmoral2) [reply factory/cluster.go:30]: sure

File: modules/install/k3s_master.sh [hunk 10]
```diff
@@ -51,7 +51,7 @@ policy_files() {
   fi
 }
 
-rhel() {
+network_manager() {
```
[the remark this answers is not in the document]
Comment (fmoral2) [reply modules/install/k3s_master.sh:54]: but what we are doing is stuff with net manager . OS config is too generic. could be any stuff.

File: pkg/assert/host.go (hunk 3 above)
Comment (fmoral2) [reply pkg/assert/host.go:12]: this func is calling the RunCmd.....to check some result.

File: pkg/template/helper.go (hunk 7 above)
Comment (fmoral2) [reply pkg/template/helper.go:61]: its too repetitive....in the func name we alreadu have TestCase, the name of the map is testCase...so no need to a var in use only in this scope, to be named like this.

Comment (fmoral2) [reply pkg/template/helper.go:61]: it will be a shadow, the map already has this name

File: pkg/template/helper.go (hunk 8 above)
Comment (fmoral2) [reply pkg/template/helper.go:63]: it will be a shadow, the map already has this name

**Pull request closed without merging**